eng/blockReflectionAttribute.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <!--
+    Providing a definition for __BlockReflectionAttribute in an assembly is a signal to the .NET Native toolchain
```

Comment:
"as a signal" ? and above

Reply:
🤷‍♂️ this is coming from buildtools.  I noticed it because without it some assemblies wouldn't have a corelib reference and CCI would barf because it couldn't decide which corlib to use.  I'll see if I can clean this up a bit.